pkg/generate/app/app_test.go

```diff
@@ -314,7 +314,7 @@ func TestSourceRefBuildSourceURI(t *testing.T) {
 	}
 }
 
-func ExampleGenerateSimpleDockerApp() {
```

Comment:
This is not functionally equivalent.  Changing the name prevents the example from running.  Given that the output isn't very interesting, you could convert it to a `Test` instead.